install.sh

```diff
@@ -194,9 +194,6 @@ info  "-y (--yes-to-all):       ${arg_y}"
 this_script="$(basename "$0")"
 export this_script
 
-export install_path="${arg_i%/}"
-info "install_path=\"${install_path}\""
-
```

Comment:
It seems that there is a problem here: Later, on line [296](https://github.com/sourceryinstitute/opencoarrays/pull/213/files#diff-3fbb47e318cd8802bd325e7da9aaabe8R296) `install_path` is de-referenced before being defined. It's unclear to me why this [only registers as an error on Linux](https://travis-ci.org/sourceryinstitute/opencoarrays/jobs/156959330#L426) but, even though [the OS X script complains about this](https://travis-ci.org/sourceryinstitute/opencoarrays/jobs/156959327#L222), it [doesn't register as an error with Travis-CI](https://travis-ci.org/sourceryinstitute/opencoarrays/builds/156959324)